server/src/services/org_environment_variables.service.ts

```diff
@@ -6,27 +6,36 @@ import { Repository } from 'typeorm';
 import { User } from '../entities/user.entity';
 import { cleanObject } from 'src/helpers/utils.helper';
 import { EncryptionService } from './encryption.service';
+import { AppsService } from './apps.service';
 
 @Injectable()
 export class OrgEnvironmentVariablesService {
   constructor(
     @InjectRepository(OrgEnvironmentVariable)
     private orgEnvironmentVariablesRepository: Repository<OrgEnvironmentVariable>,
-    private encryptionService: EncryptionService
+    private encryptionService: EncryptionService,
+    private appsService: AppsService
   ) {}
 
-  async fetchVariables(currentUser: User): Promise<OrgEnvironmentVariable[]> {
+  async fetchVariables(currentUser: User, slug: string): Promise<OrgEnvironmentVariable[]> {
```

Comment:
Shall we pass `organizationId` to the function? 
```
fetchVariables(currentUser.organizationId)
```
and for new route set app to request from `IsPublic` guard and 
```
fetchVariables(app.organizationId)
```